platform/darwin/src/MGLMultiPoint.mm

```diff
@@ -93,6 +100,22 @@ - (void)getCoordinates:(CLLocationCoordinate2D *)coords range:(NSRange)range
     }
 }
 
+- (void)updateCoordinates:(CLLocationCoordinate2D *)coords count:(NSUInteger)count
+{
+    if ([self isMemberOfClass:[MGLMultiPoint class]])
+    {
+        [[NSException exceptionWithName:@"MGLAbstractClassException"
+                                 reason:@"MGLMultiPoint is an abstract class"
```

Comment:
For the purposes of feature querying (but not annotations yet), we use this class to represent GeoJSON [`MultiPoint`s](http://geojson.org/geojson-spec.html#multipoint), which are just collections of points with no vertices connecting them. #5201 tracks cleaning up the class hierarchy so that MGLPolyline and MGLPolygon no longer inherit from MGLMultiPoint.